### ISSUE TITLE
Log analyzer is catching unrelated log entries

### ISSUE THREAD

**Issue by taoyl-ms** [Azure/sonic-mgmt#175]
Currently log analyzer is catching all error log entries in syslog and will fail the test if number of error lines is not consistent with expect amount.

Is it possible to add a limit that only to catch error from a certain docker? For example, when we are running everflow test, we want to ignore the errors from SNMP.

**Comment by stcheng** [comment]
https://github.com/Azure/sonic-snmpagent/issues/22 that's the SNMP errors.

**Comment by liatgrozovik** [comment]
I don't agree with the statement that we should ignore SNMP errors in Everflow test.
System should be error cleaned. If you have errors from SNMP while running everflow test or any other test they should be handled as a bug. if they are not bug they should not be raised as error.
